llvm/include/llvm/Transforms/IPO/FunctionImport.h

```diff
@@ -102,7 +102,39 @@ class FunctionImporter {
   /// elsewhere, typically by the in-memory ModuleSummaryIndex the importing
   /// decisions are made from (the module path for each summary is owned by the
   /// index's module path string table).
-  using ImportMapTy = DenseMap<StringRef, FunctionsToImportTy>;
+  class ImportMapTy {
+  public:
+    using ImportMapTyImpl = DenseMap<StringRef, FunctionsToImportTy>;
+
+    enum class AddDefinitionStatus {
+      NoChange,
```

Comment:
Suggest to briefly describe on what each entry mean. Optionally use examples.

Reply:
Done.